src/components/Publish/Pricing/index.tsx

```diff
@@ -67,12 +67,6 @@ export default function PricingFields(): ReactElement {
           content: <Fixed content={content.create.fixed} />
         }
       : undefined,
-    appConfig.allowDynamicPricing === 'true'
```

Comment:
Can also remove both effects in lines 32-61 from this file

Reply:
🚀